projects/plugins/boost/app/admin/deactivation-dialog.php

```diff
@@ -12,28 +12,20 @@
 <footer class="jp-plugin-deactivation__dialog__actions">
 	<button 
 		type="button"
-		class="jp-plugin-deactivation__button"
+		class="jp-plugin-deactivation__action--close jp-plugin-deactivation__button"
 		onclick="dispatchEvent(JetpackPluginDeactivation.events.close)"
 	><?php esc_html_e( 'Cancel', 'jetpack-boost' ); ?></button>
 	<button 
 		type="button"
-		class="jp-plugin-deactivation__button jp-plugin-deactivation__button--outline jp-plugin-deactivation__button--destructive"
-		onclick="dispatchEvent(JetpackPluginDeactivation.events.deactivate)"
+		class="jp-plugin-deactivation__action--deactivate jp-plugin-deactivation__button jp-plugin-deactivation__button--outline jp-plugin-deactivation__button--destructive"
 	><?php esc_html_e( 'Just Deactivate', 'jetpack-boost' ); ?></button>
 	<button 
 		type="button"
-		class="jp-plugin-deactivation__button jp-plugin-deactivation__button--destructive"
-		onclick="jpBoostDeactivateWithFeedback()"
+		class="jp-plugin-deactivation__action--deactivate jp-plugin-deactivation__button jp-plugin-deactivation__button--destructive"
+		onclick="window.open( 'https://jetpack.com/redirect/?source=jetpack-boost-deactivation-feedback', '_blank' )"
```

Comment:
Since we are also navigating to the deactivation link on the same click, the browser detects this as a popup and blocks if we mix up navigation via js and html

Reply:
Oh, that's pretty useful information to add as a comment then.